backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -537,12 +545,22 @@ let rec lambdaToSql
         | EInt8(_, v) ->
           typecheck $"Int8 {v}" TInt8 expectedType
           let name = randomString 10
-          return $"(@{name})", [ name, Sql.int8 v ], TInt8
+          return $"(@{name})", [ name, Sql.int16 (int16 v) ], TInt8
```

Comment:
was this change tot he EInt8 branch intentional?

Reply:
I believe this change was made to enable the use of the 'y' suffix in DB tests instead of using `fromInt64`